.github/workflows/user_yaml_ci.yaml

```diff
@@ -58,7 +59,10 @@ jobs:
           git checkout localbranch
         fi
 
-        # lint the files
+        # lint the files using yamllint
+        yamllint ./users/*
```

Comment:
I think it is indeed using that config file since that file is in the working directory, as suggested by https://yamllint.readthedocs.io/en/latest/configuration.html#configuration

Reply:
so that the `line-length` issues are showing as warning instead of error
but that config file did `extends: default`, which is I think why other rules got involved

Reply:
ah gotcha, it should be in this repo though so it won't work the same in repos other than commons-users

Reply:
there is a rule file in this repo also https://github.com/uc-cdis/.github/blob/master/.github/linters/.yaml-lint.yml
but it is different with the one we have in `common-users` currently, IDK if we simeply just want to replace it

Reply:
i think that one is for code... for user.yaml the simple one is fine i think 🤔 we could write the config file on the fly like we did [here](https://github.com/uc-cdis/.github/blob/680bfa610bdd877557261bc3d5ebab8ae47ccb36/.github/workflows/user_yaml_ci.yaml#L63-L79)

Reply:
it's really just about catching duplicate keys so we don't have unexpected surprises with empty dictionaries replacing dictionaries of policies users should have

Reply:
yeah IDK, sounds to me if the linter config will be specific to user yaml anyways, then keep it with the user yaml repo should be okay
but I can also refactor this to have the config included on the fly

Reply:
so basically the `key-duplicates` rule is the only one we want?

Reply:
yes pretty much

Reply:
https://github.com/uc-cdis/commons-users/actions/runs/10815466999/job/30004422471

Reply:
nice 😄